.ci/generate_redirects.jl

```diff
@@ -89,13 +95,60 @@ function create_redirect_page(; name, path)
     end
 end
 
+function create_404_page()
+    open(joinpath("webroot", "404.html"), "w") do io
+        write(io, """
+        <!DOCTYPE html>
+        <html lang="en">
+        <head>
+            <meta charset="UTF-8">
+            <meta name="viewport" content="width=device-width, initial-scale=1.0">
+            <title>Page not found</title>
+            $(style_block("#ffcbc8", "#CB3C33"))
+        </head>
+        <body>
+            <div class="centered-div">
+                <p>No page exists here.<br><br>Redirection pages for registered Julia packages follow the format packages/redirect_to_repo/SomePackage.</p>
+            </div>
+
+            <script>
+            // Get the current URL path
+            const urlPath = window.location.pathname;
+
+            // Define the regex pattern to match the URL structure
+            const pattern = /\\/packages\\/redirect_to_repo\\/([^\\/]+?)(\\.html)?\$/;
+
+            // Check if the URL matches the pattern
+            const match = urlPath.match(pattern);
+
+            if (match) {
+                const packageName = match[1]; // Extract the package name
+                const messageElement = document.querySelector(".centered-div p");
+                // Update the paragraph text
+                messageElement.innerHTML = `\
+                    There is no package \${packageName} registered in the Julia General Registry.\
```

Comment:
Would it be better to say `Example.jl` instead of just `Example`?

```suggestion
                    There is no package \${packageName}.jl registered in the Julia General Registry.\
```